docs/ABOUT.md

```diff
@@ -0,0 +1,11 @@
+# What is JavaScript?
+---
+Javascript is a scripting language used to provide dynamic and interactive content on webpages. It does not have class-oriented inheritance, but it does have prototype-oriented inheritance. Client-side JavaScript is commonly used. JavaScript provides benefits like Less server interaction, Immediate feedback to the site visitors, Increased interactivity and Richer interfaces.
```

Comment:
I'm not sure what _Client-side JavaScript is commonly used._ means here.

I really like that you're talking about the benefits. How about rewording the benefits so that they're more conversational, and putting them right after the first sentence?